pages/index.vue

```diff
@@ -90,11 +90,13 @@ export default {
   computed: {
     upcomingEvents: (vm) =>
       vm.events.filter(
-        (event) => new Date(event.sort_by_date).setHours(24) > Date.now()
+        (event) =>
+          new Date(event.sort_by_date + ' 00:00:00').setHours(24) > Date.now()
```

Comment:
Not a required change , just a proposal: `${event.sort_by_date} 00:00:00` instead of `event.sort_by_date + ' 00:00:00'`
Same below if you decide to apply this.

Reply:
Thanks! Yaah! That cross my mind, but I went the _lazy_ "just add it at the end" way this time :P